sourceme.sh

```diff
@@ -8,26 +8,45 @@ alias apidev="OPTIC_DAEMON_ENABLE_DEBUGGING=yes OPTIC_DEVELOPMENT=yes OPTIC_UI_H
 alias apistage="OPTIC_DAEMON_ENABLE_DEBUGGING=yes $OPTIC_SRC_DIR/workspaces/local-cli/bin/run"
 alias cidev="$OPTIC_SRC_DIR/workspaces/ci-cli/bin/run"
 alias agentdev="$OPTIC_SRC_DIR/workspaces/agent-cli/bin/run"
+
+optic_export_env() {
+  ENV_FILE=$1
+  if [ ! -f "$ENV_FILE" ]
+  then
+    echo "could not find env file '$ENV_FILE'"
+    exit 1
+  fi
+
+  VARS_TO_EXPORT=$(grep -v '^#' "$ENV_FILE")
+  VARS_WITHOUT_WHITESPACE=$(echo $VARS_TO_EXPORT | sed s/\n// | sed s/\ //)
```

Comment:
not sure i'm following why we strip newlines?

Reply:
The grep can emit blank lines, but perhaps the echo will remove them. Perhaps the better check is to see if there are any alphanumeric characters at all?